incubator/hnc/internal/reconcilers/hierarchy_config.go

```diff
@@ -575,19 +593,19 @@ func (r *HierarchyConfigReconciler) writeHierarchy(ctx context.Context, log logr
 	}
 	exists := !inst.CreationTimestamp.IsZero()
 	if !exists && isDeletingNS {
-		log.Info("Will not create singleton since namespace is being deleted")
+		log.Info("Will not create hierarchyconfiguration since namespace is being deleted")
 		return false, nil
 	}
 
 	stats.WriteHierConfig()
 	if !exists {
-		log.Info("Creating singleton on apiserver", "conditions", len(inst.Status.Conditions))
+		log.Info("Creating hierarchyconfiguration", "conditions", len(inst.Status.Conditions))
 		if err := r.Create(ctx, inst); err != nil {
 			log.Error(err, "while creating on apiserver")
 			return false, err
 		}
 	} else {
-		log.Info("Updating singleton on apiserver", "conditions", len(inst.Status.Conditions))
+		log.V(1).Info("Updating singleton on apiserver", "conditions", len(inst.Status.Conditions))
```

Comment:
Why this is v1, is it too verbose? And r/singleton/hierarchyconfiguration?

Reply:
It's quite verbose and doesn't tell a cluster admin anything useful - it's only useful to developers.

I left the term "singleton" in for the developer-only messages, it's only the externally facing messages where I updated everything.

Reply:
sg